IOPool/Streamer/interface/FRDEventMessage.h

```diff
@@ -15,6 +15,18 @@
  * 14-Nov-2013 - RKM  - Added event size, adler32 and padding size (version #3)
  * 15-Oct-2014 - WDD  - Event number from 32 bits to 64 bits (version #4)
  * 01-Apr-2015 - SM   - replaced adler32 with crc32c which is accelerated in SSE 4.2 (version #5)
+ * 22-Sep-2020 - SM   - reused high version 16-bits for event info via flags (version #6)
+ *
+ *
+ * Version 6 Format:
+ *   uint16 - format version number
+ *   uint16 - eventf flags
```

Comment:
Just to make sure that I understood correctly. From `FRDEventMsgView` and `RawEventOutputModuleForBU ` I understand that the flags are in the higher 16-bit part and the version number in the lower 16-bit part of the first 32-bit word. But here the version number becomes before the flags. And that is correct because of the (little) endianness, i.e. reading the first 16 bits gives the version number, and the second 16 bits gives the flags.

By the way, is there a typo?
```suggestion
 *   uint16 - event flags
```

Reply:
The intention is to have 16-bit version before the flag word in the file. I will fix the code to always order the two variables in this respect.

In practice, the handling of the format hasn't been endian safe.  For example, with little-endian first byte of the header will be end up having the the version number, in big-endian it will be at later position. However, there is currently no use, AFAIK, for this format on big-endian architectures (only Intel/AMD used with it in DAQ for the time being).

I'll correct also the typo you mentioned.

Reply:
Power and ARM builds are also little endian.

Reply:
👍

Reply:
For a bit more clarity in the code, I switched to using 16-bit type pointers to write or read version and flags. If there is no remaining issue, I'll mark this resolved.